### PR TITLE
remove several test scripts from packages

### PR DESCRIPTION
**Pull request by jsteemann**
### Scope & Purpose

Remove some JavaScript utility files used for testing from the packages, by moving them into a separate directory and excluding it from packaging.

- [x] Bug-Fix for *devel-branch* (i.e. no need for backports?)
- [x] Strictly *new functionality* (i.e. a new feature / new option, no need for porting)
- [ ] The behavior in this PR can be (and was) *manually tested* (support / qa / customers can test it)
- [x] The behavior change can be verified via automatic tests

### Testing & Verification

This change is already covered by existing tests, such as *all tests*.

http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11304/

### CODE REVIEW

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/11443/

**Comment by jsteemann** [comment]
Tests blue except "no space left on device" for MacOS.

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-packages/15/

File: cmake/InstallArangoDBJSClient.cmake [hunk 1]
```diff
@@ -10,6 +10,8 @@ install(
     ${CMAKE_INSTALL_DATAROOTDIR_ARANGO}/${ARANGODB_JS_VERSION}
   FILES_MATCHING
     PATTERN "*.js"
+  REGEX "^.*/js/client/modules/@arangodb/testsuites/" EXCLUDE
+  REGEX "^.*/js/client/modules/@arangodb/testutils/" EXCLUDE
```
[the remark this answers is not in the document]
Comment (jsteemann) [reply cmake/InstallArangoDBJSClient.cmake]: Ok, I applied these changes.

**Comment by jsteemann** [comment]
@KVS85 : shall we merge it eventually?

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-ANY-packages/16/

**Review by KVS85 — APPROVED** [review]

**Pull request merged**